benches/bench.rs

```diff
@@ -22,8 +22,8 @@ fn release_fixture_path() -> PathBuf {
 }
 
 fn with_file<F: FnOnce(&object::File)>(target: &path::Path, f: F) {
```

Comment:
Need to open the file here (like you did in the other conversions).

Reply:
Right, missed this part.

Reply:
Should be fixed now.